### PR TITLE
feat(ui): format print page and add print button to datatable

### PR DESCRIPTION
**Pull request by maierthomas**
- Change print layout for all pages
- Add print button to data tables to export all data
- Catch CTRL+P and print all entries 

closes sw360/sw360portal#502

### CODE REVIEW

**Comment by mcjaeger** [comment]
there is a problem with added java scripts - the UI elements Proejcts listing and component listing do not build up properly (empty table) with following error from the console:

image

File: frontend/sw360-portlet/src/main/webapp/html/admin/vendors/view.jsp [hunk 1]
```diff
@@ -68,7 +68,7 @@
     AUI().use('liferay-portlet-url', function () {
         var PortletURL = Liferay.PortletURL;
 
-        require(['jquery', 'utils/includes/quickfilter', 'modules/confirm', /* jquery-plugins: */ 'datatables', 'jquery-confirm'], function($, quickfilter, confirm) {
+        require(['jquery', 'utils/includes/quickfilter', 'modules/confirm', /* jquery-plugins: */ 'datatables.net', 'datatables.net-buttons', 'buttons.print', 'jquery-confirm'], function($, quickfilter, confirm) {
```
Comment (alexbrdn) [inline frontend/sw360-portlet/src/main/webapp/html/admin/vendors/view.jsp]: Why is it necessary to change `'datatables'` to `'datatables.net'`? Please explain.

Comment (maierthomas) [reply frontend/sw360-portlet/src/main/webapp/html/admin/vendors/view.jsp]: The js for the print functionality itself requires the following scripts
- datatables.net
- datatables.net.buttons
define( ['jquery', 'datatables.net', 'datatables.net-buttons'], function ( $ ))

But I found a solution to use the datatable require statements as before.
We can map the datatables.net name to datable by using the "map" function in our require.js file.

  map: {
            '*': {
                'datatables.net': 'datatables',
                'datatables.net-buttons': 'datatables_buttons'
            }
        }

**Comment by alexbrdn** [comment]
With this new version, I don't get the 404 error anymore. Projects and components tables are loaded and displayed correctly.

**Comment by mcjaeger** [comment]
so can I change to "needs review"?

**Comment by alexbrdn** [comment]
done that

**Comment by mcjaeger** [comment]
just a cut off example:
[Scaring Daring Components - SW360.pdf](https://github.com/sw360/sw360portal/files/1306171/Scaring.Daring.Components.-.SW360.pdf)

(tested on plain HP LaserJet PCL5 Driver here in offer)

**Comment by maierthomas** [comment]
- Add print button to project clearing status and component release overview
(remove action icons if you hit the print button for project clearing and component release overview).

- Fix cut off issue for print mode portrait.

Examples:
Chrome landscape mode
chrome_landscape

Chrome portrait
chrome_portrait

Firefox landscape
firefox_lanscape

Firefox portrait
firefox_portrait

**Comment by mcjaeger** [comment]
work in firefox now

**Pull request merged**